amr-wind/incflo.cpp

```diff
@@ -372,6 +378,10 @@ void incflo::init_physics_and_pde()
         if (activate_overset) {
             m_sim.activate_overset();
         }
+
+        // Get number of advection iterations
+        pp.query("advection_iterations", m_adv_iters);
+        m_adv_iters = std::max(1, m_adv_iters);
```

Comment:
do we want to silently do this? Or abort on an invalid value? Or warn that we are doing this?

Reply:
Got rid of this.